test/unit/rules/no-negated-condition-test.js

```diff
@@ -3,7 +3,7 @@ import generateRuleTests from '../../helpers/rule-test-harness.js';
 generateRuleTests({
   name: 'no-negated-condition',
 
-  config: true,
+  config: { simplifyHelpers: true },
```

Comment:
```suggestion
  config: true,
```

Let's just test the default behavior here. Then in individual test cases, we can test the explicit option behavior from: `{ simplifyHelpers: true }` and `{ simplifyHelpers: false }`.